src/Apps/Collect2/Routes/Collection/Components/CollectionsHubRails/FeaturedCollectionsRails/index.tsx

```diff
@@ -92,6 +87,26 @@ export const FeaturedCollectionsRails: React.FC<Props> = ({
         }}
         onArrowClick={() => trackArrowClick()}
       />
+    )
+  }
+
+  return (
+    <FeaturedCollectionsContainer>
+      <Serif size="5" mt={3}>
+        {name}
+      </Serif>
+      <Media lessThan="md">
+        {renderCarousel({ xs: true, sm: true, xl: false })}
```

Comment:
These (and following) can omit falsy values, as undefined will resolve to `false`.